### ISSUE TITLE
GNA.org is down: package download impossible through Ansible Role

### ISSUE THREAD

**Issue by arondeparon** [ansiblebit/wkhtmltopdf#8]
It seems that gna.org is dead, so an alternative download location should be supplied.

https://github.com/ansiblebit/wkhtmltopdf/blob/master/defaults/main.yml#L17

**Comment by steenzout** [comment]
@ArondeParon you can using `wkhtmltopdf_package_download_url` variable.

**Comment by steenzout** [comment]
give me 1h and I'll patch the default value.

in the meantime you can use the variable above and point it to: https://downloads.wkhtmltopdf.org/0.12/0.12.1/ (or whatever version you are looking for).

thank you for reporting this.

**Comment by steenzout** [comment]
```
TASK [wkhtmltopdf : package | download] ****************************************
fatal: [precise64.vagrant.dev]: FAILED! => {"changed": false, "dest": "/home/vagrant/wkhtmltox-0.12.1_linux-precise-amd64.deb", "failed": true, "msg": "Request failed", "response": "Request failed: <urlopen error timed out>", "state": "absent", "status_code": -1, "url": "http://download.gna.org/wkhtmltopdf/0.12/0.12.1/wkhtmltox-0.12.1_linux-precise-amd64.deb"}
```

**Comment by steenzout** [comment]
@ArondeParon patched on [2.16.8](https://github.com/ansiblebit/wkhtmltopdf/releases/tag/2.16.8).

I ran tests locally and on travis-ci and everything looks good.
if you still see a problem let me know.

_please star the repo if you use it; thank you!_
